src/app/screens/admin/components/projects/projects.component.ts

```diff
@@ -1,20 +1,73 @@
-import { Component, OnInit } from "@angular/core";
-import { DomSanitizer } from "@angular/platform-browser";
-import { MatIconRegistry } from "@angular/material/icon";
+import { Component, OnInit } from '@angular/core';
+import {FormGroup, FormControl} from '@angular/forms';
+import { ProjectService } from 'src/app/services/project.service';
+// import { ToastService } from 'ng-uikit-pro-standard';
+
+declare var $: any;
```

Comment:
no more jquery thanks, please bind a class in your view to an angular property and style the class as appropriate